package.json

```diff
@@ -41,10 +41,10 @@
     "lint": "npx npm-run-all -p lint:*"
   },
   "dependencies": {
-    "fast-glob": "^2.2.3",
```

Comment:
don't include dependency updates with other changes, it's too disruptive to the release flow